extensions/wikia/PhalanxII/classes/PhalanxPager.class.php

```diff
@@ -70,88 +85,108 @@ function getQueryInfo() {
 		return $query;
 	}
 
+	/**
+	 * Returns the name of the table field used for indexing and sorting if PHP fallback is used
+	 * @return string table field used for indexing and sorting if PHP fallback is used
+	 */
 	function getIndexField() {
 		return 'p_timestamp';
 	}
 
+	/**
+	 * Returns the opening tag for the HTML element wrapping the results
+	 * @return string Opening HTML tag for wrapper
+	 */
 	function getStartBody() {
-		return Html::openElement( 'ul', array( "id" => 'phalanx-block-' . $this->pInx . '-' . $this->id . '-stats' ) );
+		return Html::openElement( 'ul', [ "id" => 'phalanx-block-' . $this->pInx . '-' . $this->id . '-stats' ] );
 	}
 
+	/**
+	 * Returns the closing tag for the HTML element wrapping the results
+	 * @return string Closing HTML tag for wrapper
+	 */
 	function getEndBody() {
 		return Html::closeElement( 'ul' );
 	}
 
+	/**
+	 * Returns the boilerplate content to be displayed if no results are available
+	 * @return string HTML to display if no results are available
+	 */
 	function getEmptyBody() {
-		return Html::element('div', array('class' => 'error'), wfMsg('phalanx-no-results'));
+		return Html::element( 'div', [ 'class' => 'error' ], $this->msg( 'phalanx-no-results' )->text() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "class" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)